src/style.css

```diff
@@ -0,0 +1,83 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+body {
+  box-sizing: border-box;
+}
```

Comment:
better give `body` some class, and add styles to this class 😉

Reply:
ok, thanks